frontend/src/pages/AGEntryPage.tsx

```diff
@@ -3,10 +3,38 @@
  */
 
 import React from 'react';
+import './AGEntryPage.css';
+import Ballbild from '../images/Wahlpflicht.png';
 
 function AGEntryPage() {
```

Comment:
diese Komponente ist eigentlich die Komponente wo alle AG's die für einen Schüler verfügbar sind angezeigt werden, also dass was Max gerade macht.

Du kannst deins ja in eine extra Komponente packen, sowas wie "DetailAGInfoPage". Im Rounting in der App.tsx muss für diese dann eine extra Seite angelegt werden